spec/features/user_adds_a_new_drink_spec.rb

```diff
@@ -1,24 +1,25 @@
 require "rails_helper"
-feature "user adds a new drink" do
-# As a user,
-# I want to be able to create a new drink,
-# so that it can be reviewed and viewed in
-
-# Acceptance Critieria:
-# [X] When user creates a new drink,
-#     drink page displays the new drinks information
-# [X] Successfully adding a drink displays a
-#     message stating a new drink has been created
-# [X] Must specify a name, if name is not specified display error message
-# [X] Must specify either alcoholic or non alcoholic,
-#     if name is not specified display error message
-# []  If alcoholic, a list of alcohols should apear,
-#     allowing the user to pick several
-# [X] I can only input drink names that have at least 3 letters.
-# [X] I must input a description
-# [X] I must be signed in to be able to add a drink
-# [X] If drink is alcoholic, alcohol types must be specified
-# []  Can optionally provide a photo
+feature "user adds a new drink", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.